crabs/detection_tracking/inference_model.py

```diff
@@ -96,27 +105,35 @@ def load_video(self) -> None:
         if not self.video.isOpened():
```

Comment:
we don't seem to use `self.trained_model = self.load_trained_model()` in load_video right? Should we move it to __init__?